flux-middle/src/rustc/ty.rs

```diff
@@ -95,6 +95,7 @@ pub enum TyKind {
     Tuple(List<Ty>),
     Uint(UintTy),
     Slice(Ty),
+    Ptr(Ty, Mutability),
```

Comment:
Same here, if it's called a `RawPtr` in rustc we should also call it a `RawPtr` here
```suggestion
    RawPtr(Ty, Mutability),
```